cmd/statusd/main.go

```diff
@@ -157,6 +157,7 @@ Examples:
   statusd -cli          # enable connection by statusd-cli on default port
 
 Options:
-`)
+`
+	fmt.Fprintf(os.Stderr, usage) // nolint: gas
```

Comment:
I wonder why do we print usage to stderr?

Reply:
@adambabik I think it's a common UNIX practice and helpful when you pipe commands through the chain of pipes: if the program is used incorrectly, the user will see the error instead of "usage" text piped into the next command.
Not our case, of course, but I believe that's the reason for using stderr for usage output.

Reply:
It makes perfect sense 👍  Hopefully, we also return a non-zero exit code.